### ISSUE TITLE
[4.0][a11y]Insufficient contrast in the Quickicon module

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20993]
The contrast between white text and green background in "Joomla is up to date" & "Extension are ap to date" icons is  3.46:1. It should at least amount to: 4.5:1, e.g #ffffff/#148914
See: [Contrast minimum](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

**Comment by brianteeman** [comment]
For things like this it is best to wait a week or so for the latest version of the template to be merged in OR to do the test against https://github.com/joomla/40-backend-template/

**Comment by brianteeman** [comment]
I am going to close this at this time as it is completely different in the latest template
